src/components/ProfileDashboard.js

```diff
@@ -0,0 +1,466 @@
+                firstName: firstName,
+                lastName: lastName,
+                aboutMe: aboutMe,
+                yearSemester: yearSemester,
+                program: program,
+                interest: interest,
+                coop: coop,
+                userID: userID,
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        console.log("Profile:", body);
+        return body;
+    }
+
+    const handleApiAddSubmission = () => {
+        callApiAddSubmission()
+            .then(res => {
+                console.log("callApiAddSubmission returned: ", res)
+            })
+    }
+
+    // Create states to store any existing profile information about the User
+    const [existingAboutMe, setExistingAboutMe] = React.useState();
+    const [existingYear, setExistingYear] = React.useState();
+    const [existingProgram, setExistingProgram] = React.useState();
+    const [existingInterest, setExistingInterest] = React.useState();
+    const [existingCoop, setExistingCoop] = React.useState();
+
+    // Display the user's profile 
+    // Obtain the appropriate fields 
+    // Load the profile if it is not empty
+    const [loadProfile, setLoadProfile] = React.useState();
+
+    // Inserts the User ID to retrieve their most recent profile 
+    const callAPIUserProfile = async (userID) => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/getUserProfile`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+                //authorization: `Bearer ${this.state.token}`
+            },
+            body: JSON.stringify({
+                userID: userID,
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        console.log("Profile:", body);
+        return body;
+    }
+
+    // Parses through the data for the user if there is existing data
+    const handleAPIUserProfile = (userID) => {
+        callAPIUserProfile(userID)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+
+                if (parsed.length !== 0) {
+                    setExistingAboutMe(parsed[0].aboutMe);
+                    setExistingYear(parsed[0].yearSemester);
+                    setExistingProgram(parsed[0].program);
+                    setExistingInterest(parsed[0].interest);
+                    setExistingCoop(parsed[0].coop);
+                }
+            })
+    }
+    
+    return (
+        // Displays the AppBar, Drawer, the Title, the fields and conditionals for the SAVE/EDIT button. 
+        <div className={classes.root}>
+            <AppBar position="fixed" className={classes.appBar}>
+                <Toolbar>
+                    <Typography variant="h3" noWrap>
+                        Profile
```

Comment:
Please use the `NavigationBar` component here. Add a route to the profile section that routes the user to this page. 

![image](https://user-images.githubusercontent.com/57375371/220451177-2fd4b31f-359b-49a7-afa6-7c668d8ed16d.png)

There should be an additional option here, something like "My Profile".